### PR TITLE
feat: drop python 3.2, 3.3, support 3.6, 3.7

### PR DESCRIPTION
**Pull request by joshuarli**
Tests pass in a python3.7 venv, so the rest should just work. Unfortunately, travis xenial doesn't have py32, 33, so I've dropped support for that. I don't think legacy (trusty) travis images have py37.

I've also pinned dependencies so packages that are dependent on toronado don't have to pin toronado's dependencies themselves. To my knowledge, these are all good versions that work with py27, py34-37. We also currently run this in sentry prod:

```
toronado==0.0.11
  - cssselect [required: Any, installed: 1.0.3]
  - cssutils [required: Any, installed: 1.0.2]
  - lxml [required: Any, installed: 4.3.3]
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,14 +4,14 @@
 
 
 install_requires = [
-    'cssselect',
-    'cssutils',
-    'lxml',
+    'cssselect==1.0.3',
```
Comment (mattrobenolt) [inline setup.py]: Strong -1 to pinning these strictly. If anything, you need to pin a range that's api compatible.

Comment (joshuarli) [reply setup.py]: oof, that's gonna take a while to get right... any tips? Or should I just not pin anything, leaving it up to toronado users?

Comment (mattrobenolt) [reply setup.py]: Yeah, I'd leave it just unpinned. Does latest versions not work or something? What prompted pinning these?

Comment (joshuarli) [reply setup.py]: I mainly wanted to unpin them in sentry, by moving the responsibility to toronado. I still think ideally toronado should pin these, but yeah, I agree with range pins being the best. Although it takes quite some time to get right, so.... not worth it.

**Pull request closed without merging**